checker/src/main/java/org/checkerframework/checker/guieffect/GuiEffectTypeFactory.java

```diff
@@ -420,126 +441,76 @@ public AnnotatedTypeMirror getAnnotatedType(Tree tree) {
         ExecutableElement polyOverride = null;
```

Comment:
As a minor comment, I think the above 3 variables would be better named *Overridden rather than *Override, since they contain a method of the given effect that is overridden.

Reply:
Good suggestions, I've renamed them.